### DISCUSSION TITLE
How to add LORAs correctly?  I didn't understand very well.

### DISCUSSION THREAD

**Discussion by WBV-CWB** [hako-mikan/sd-webui-regional-prompter#242] [Q&A]
How to add LORAs correctly?  I didn't understand very well.

Well, i have some doubts, and... I can't to find any youtube video about this.

WELL...

I have:
anime screencap, anime coloring, (2girls different ages:1.1),forest, nature, cherry blossoms, private garden,
ADDBASE
 <lora:Sarada:0.5>, loli, ((zipper open)),((bra)), black thighhighs,  black hair,short hair, (dress:1.1), bare shoulders, arm warmers,( shorts:0.8), glasses,red-framed eyewear, small breasts, thigh, black eyes,
ADDCOL
 <lora:Sakura[Boruto]V2:0.5>, haruno sakura, ((forehead mark)), short hair, pink hair, red hairband, red sleeveless dress, white pants, pants pull, clothes pull, undressing, navel, groin, bracelet,  medium breast ,(((wide hips))), toned body, detailed green eyes, thong, thong panties, pink panties, looking at viewer,

Is that the right way? Seems like the LORAs are not applied correctly. Render looks distorted the characters are being fused during the generation or any worse thing. 

I have:
Regional Prompter ACTIVE
Generation mode LATENT (Everyone I see talking about it will tell to use this, but I've tried with Attention too)
Base Ratio 0.2
Common prompt checked
Split Mode HORIZONTAL
Divide Ratio 1,1

This my SD print:
![1](https://github.com/hako-mikan/sd-webui-regional-prompter/assets/70992721/0c7816dd-2661-4140-8441-fa2865671da9)

![2](https://github.com/hako-mikan/sd-webui-regional-prompter/assets/70992721/8e61ccac-edaa-4c40-b159-8f52045a7028)

![3](https://github.com/hako-mikan/sd-webui-regional-prompter/assets/70992721/94b2fd24-eb14-4ccb-9b6a-82334dfc4fc3)

![4](https://github.com/hako-mikan/sd-webui-regional-prompter/assets/70992721/ff8ad946-b93f-400e-b1e4-c2bfab367aea)

And this, I would like to know what is this? Does this help with something for the better in the image of the generation?

![5](https://github.com/hako-mikan/sd-webui-regional-prompter/assets/70992721/3576bf00-7382-4b00-82fc-b4d541ce0daf)

Thanks in advance

**Comment by hako-mikan** [comment]
Please refer to the information below. Given your usage, I think COMMON would be better than BASE.
https://github.com/hako-mikan/sd-webui-regional-prompter/issues/221#issuecomment-1731654542

**Comment by redstoneguy10ls** [comment]
im having the same problem here
